src/components/sbb-toggle-check/readme.md

```diff
@@ -0,0 +1,70 @@
+
+`<sbb-toggle-check>` provides the same functionality as a native `<input type="checkbox">`
+enhanced with the SBB Design.
+
+```html
+<sbb-toggle-check
+  value="single-checkbox"
+  disabled="false"
+  checked="false"
+>
+  Example
+</sbb-toggle-check>
+```
+
+## Label
+
+The label is provided as the content to the `<sbb-toggle-check>` element.
+It can be displayed after or before the toggle-check.
+
+If you don't want the label to appear next to the toggle-check, you can use
+[`aria-label`](https://www.w3.org/TR/wai-aria/states_and_properties#aria-label) or
```

Comment:
This is incorrect, as we use the properties `accessibility-*` to place the actual `aria-*` attributes on the checkbox.